css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-default-size: 32px !default;
+$persona-font-size: $font-size-7 !default;
+$persona-gap-size: $layout-1 !default;
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+$persona-dimension-font-size-ratio: 0.45;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-font-size;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: $persona-default-size;
+		height: $persona-default-size;
+		border-radius: $border-radius-rounded;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-default-size * $persona-dimension-font-size-ratio;
+		overflow: hidden;
+
+		img,
+		svg {
+			display: block;
+			width: 100%;
+			height: auto;
+			max-height: 100%;
+		}
+
+		span {
+			line-height: 1;
+			text-transform: uppercase;
+		}
```

Comment:
I'd like to propose we don't support styled text for avatars at all. Instead, when we want to use initials as a fallback avatar, I think we should encourage [an approach we're already using in Q&A for fallback avatar generation](https://dev.azure.com/ceapex/Engineering/_git/docs-ui/pullrequest/757907?_a=files), where we generate an SVG as a data URI, and then use that data URI as an `img`'s `src`.

Possibly, a utility for generating that data URI could even exist in Atlas's scripts - I could go either way on that front.

Using styled text for fallback avatars...
- Can cause problems if the user has changed their browser or device-level base font size
- Means styled text avatars will be modified, probably unexpectedly, by things like operating system forced colors mode/contrast themes
- Introduces variation in markup, which complicates rendering logic and means different "avatars" announce differently to assistive technologies

Reply:
I was looking into how we currently treat the initials, and saw that mostly those are generated SVGs (Yay!). Though I though it's good to have that option just in case, as a framework we should consider other cases besides learn.

But I was actually looking for your feedback on what would be best to do with initials markup and if we should assign it `aria-label="user name"` or something else so it announced the same way as the image avatar 🤔

Reply:
`aria-label` doesn't work on `span`s out of of the box, since `span`s have `generic` roles by default.

_If_ we did want to both support `aria-label` and ensure announcements are consistent with those of actual image avatars, we could use `role="img"`:

```html
<span role="img" aria-label="Avatar for Jane Doe">
    JD
</span>
```

Reply:
@wibjorn @homing1 any thoughts on this?

Reply:
Fine to go with using SVGs only.

Reply:
I was about to remove the letters version but realized we have places in learn where it is used.  Specifically in facepile component, which is gonna be replaced with persona.

So I applied the attributes Ben suggested so both versioned are announced the same way by screen readers.

Reply:
I don't think we need to support that old stuff. It should be relatively trivial to produce an SVG with initials anyway.

Reply:
Sounds good, I will remove the initials version completely then

Reply:
The initials version has been removed.